config/devel.yaml

```diff
@@ -7,8 +7,8 @@ PLAINTEXT_VARIABLES:
     SIERRA_DB_NAME: iii
     REDSHIFT_DB_NAME: qa
     ENVISIONWARE_BATCH_SIZE: 20
-    S3_BUCKET: bic-poller-states-qa
-    S3_RESOURCE: pc_reserve_poller_state.json
+    S3_BUCKET:
```

Comment:
why are these blank

Reply:
devel doesn't use the cache and I thought it would be confusing to keep the unused QA cache variables in the config